### PR TITLE
final

### PR DESCRIPTION
**Pull request by alexprokopov1992**

### CODE REVIEW

File: Bank/Bank/DataBase.h [hunk 1]
```diff
@@ -59,11 +61,92 @@ class DataBase
     void averageTransactionsPerDay()
     {
         //TODO: output average transactions amount per day
+        if (clients.empty() == 0)
+        {
+            float totalRefill = 0;
```
Comment (miklealex) [inline Bank/Bank/DataBase.h:66]: Average per day means we need to calculate total transactions amount per each day and then calculate the average of those values.
Not just for today.

File: Bank/Bank/DataBase.h [hunk 2]
```diff
@@ -59,11 +61,92 @@ class DataBase
...
+            float maxAmount = 0;
+            for (const auto& client : clients)
+            {
+                auto history = HistoryManager::loadHistoryForClient(client.second.id);
+                for (const auto& it : history)
+                {
+                    if (maxAmount < it.amount)
+                    {
+                        maxAmount = it.amount;
+                        biggest.resize(0);
```
Comment (miklealex) [inline Bank/Bank/DataBase.h:130]: clear() instead of resize(0)

**Review by miklealex — CHANGES_REQUESTED** [review]

File: Bank/Bank/DataBase.h (hunk 1 above)
Comment (alexprokopov1992) [reply Bank/Bank/DataBase.h:65]: Changed

File: Bank/Bank/DataBase.h (hunk 2 above)
Comment (alexprokopov1992) [reply Bank/Bank/DataBase.h]: changed

**Pull request closed without merging**